config/index.js

```diff
@@ -49,6 +49,9 @@ config.mailgunApiKey = process.env.MAILGUN_API_KEY;
 // RECAPTCHA SECRET (ONLY FOR PRODUCTION)
 config.captchaSecret = process.env.CAPTCHA_SECRET;
```

Comment:
@maltejur We only had `CAPTCHA_SECRET` before. I figured we would be the field we used.

Do we need both? You might consider using `captchaSecret` in your code if this refers to the same secret.

Reply:
No there are two, `CAPTCHA_SECRET` (the captcha secret, gets used by the server) and `CAPTCHA_TOKEN` (the captcha public key, gets sent to the client). I had the `CAPTCHA_TOKEN` hardcoded in the file before, but was asked to do it this way.

Reply:
Gotcha, I misunderstood before. I will change the K8s deployment accordingly. Standby.